### PR TITLE
Simplified Chinese version

### PR DESCRIPTION
**Pull request by ben7th**
added two files:

README.zh-cn.md
glossary.zh-cn.md

### CODE REVIEW

**Comment by jboner** [comment]
Thank you very much. We will try to roll this out as soon as possible. 
@rasummer - Can you take a look at this when you get the time?

File: glossary.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* [复制](#复制)
+* [资源](#资源)
+* [可扩展性](#可扩展性)
+* [系统](#系统)
+* [用户](#用户)
+
+## <a name="异步的"></a>异步的
+牛津词典把“异步的”定义为**“不在同一时刻存在或发生的”**。在本宣言的上下文中，我们想表达的意思是：对一项请求的处理过程可以发生在任意时间点，有时候请求已经从客户端发送到服务端了，其处理过程还未开始。这时候，客户端无法直接观察到这项请求在服务端的执行过程，也无法与执行过程保持同步。作为异步的反义词，同步是指客户端暂停执行，直到服务端已经把请求处理完毕，才继续自己的执行过程。
+
+## <a name="背压"></a>背压
```
Comment (zzz6519003) [inline glossary.zh-cn.md]: 这词牛

File: README.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+响应式宣言
```
Comment (zzz6519003) [inline README.zh-cn.md]: 觉得反射式更好   因为有 响应式设计这么一说

Comment (pimgeek) [inline README.zh-cn.md]: @zzz6519003 你说的有道理，我的直觉中 Reactive 可以翻译为“反应式”（反射式或许也可以，我不确定），而不是“响应式”。
我请这个标题的译者 @ben7th 来参与讨论 ，看看他是否也赞同你的译法。 ：P

Comment (pimgeek) [inline README.zh-cn.md]: @ben7th 两周内没有回应，此修改对他的影响没有特别大，我已基于上述的协商，统一改为**反射式** ，并重新提交 Pull Request

Comment (ben7th) [inline README.zh-cn.md]: @zzz6519003 谢谢建议，不过我是这么考虑的：

1. 现在网站设计有响应式设计一说，但是我反倒觉得他叫的那个名字不准确啦。。我更倾向于把“根据设备屏幕大小自动变化布局样式的网站设计”称为“自适应设计”。

2. 如果说是大家已经习惯了“响应式设计”是说网站设计的，那同理来说“反射”也很容易因为人的习惯而联想到元编程。况且从这种架构设计和编程思想来说，他强调的是流式处理，一个要点就是每个 stream 的  subscriber 去“响应”流的变化而做出后续处理，而并非强调“一定要 response 什么东西回去”。

  反射这个词，比如神经元反射，我感觉更像那种有来有回的信息传递，而 reactive 的编程思想主要还是逐层单向流动居多。

3. 目前在各个搜索引擎搜索“响应式编程”返回的符合期望的结果数目要比“反射式编程”要多。也就是说前者可能更“深入人心”一些。

所以我个人觉得响应式这个译法是合理的。

Comment (zzz6519003) [inline README.zh-cn.md]: here's your beer :beer:

**Comment by rasummer** [comment]
@jboner @pimgeek @zzz6519003 

I'm back in the loop and working on the manifesto website. Sorry for the delay, I was unavailable for a while. Are we going with this translation as opposed to the more recent #83. If so I'll merge this one and  close the #83.

**Comment by pimgeek** [comment]
@rasummer Thanks for your feedback! 

Please note that @zzz6519003 has closed his #80 PR, and #83 is created by @xiangtao123. 

I suggest you merge the PRs in **Chronological order**. ：）

**Comment by rasummer** [comment]
@pimgeek the simplified Chinese version has been added to the website. 

There are a few additional optional UI elements that need translating that aren't part of this repo. Please feel free to open a PR on the website repo and translate the following file. https://github.com/reactivemanifesto/website-manifesto/blob/master/conf/messages.zh-CN

**Comment by pimgeek** [comment]
@rasummer Thank you SO MUCH for your help to make our translation accessible by Chinese readers! I will try to translate messages.zh-CN asap.  :+1:

**Comment by rasummer** [comment]
@pimgeek 

Happy to do so.  If you see any mistakes on my part, please feel free to submit fixes directly in the website manifesto repo.

**Pull request merged**